src/util/worker/mod.rs

```diff
@@ -161,6 +161,11 @@ impl<T: Display> Scheduler<T> {
     pub fn is_busy(&self) -> bool {
         self.counter.load(Ordering::SeqCst) > 0
     }
+
+    /// Get the count of pending task.
+    pub fn task_count(&self) -> usize {
+        self.counter.load(Ordering::SeqCst)
```

Comment:
maybe we can use crossbeam sender len directly and remove the counter.

Reply:
I will do it in another little PR.